src/hoi4_world/countries/hoi4_country_converter.cpp

```diff
@@ -174,6 +174,40 @@ std::vector<hoi4::EquipmentVariant> DetermineActiveVariants(const std::vector<ho
    return active_variants;
 }
 
+
+std::set<std::string> ConvertLaws(const std::set<std::string>& vic3_laws, std::string_view ideology)
+{
+   std::set<std::string> hoi4_laws;
+
+   // civilian economy with export focus is the default
+   hoi4_laws.emplace("civilian_economy");
+   hoi4_laws.emplace("export_focus");
+
+   // but fascist countries change it up
+   if (ideology == "fascism")
+   {
+      hoi4_laws.clear();
+      hoi4_laws.emplace("partial_economic_mobilisation");
+      hoi4_laws.emplace("limited_exports");
+   }
```

Comment:
nit: This seems really easy to accidentally mess up in the future and end up with multiple different economy types at the same time.
We may want (either now or in the future) to group by law, or do some check at the end to make sure we only have 1 economy type/mobilization type/etc.